files/default/upgrade-controller.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# Steps taken primarily from https://www.rdoproject.org/install/upgrading-rdo-2/
+# Specifically from this version for Newton https://git.io/vQGev
+
+set -ex
+
+# Disable Chef temporarily
+rm -f /etc/cron.d/chef-client
+
+# Stop and disable openstack-ceilometer-api.service as it will be served via
+# httpd now
+systemctl stop openstack-ceilometer-api.service
+systemctl disable openstack-ceilometer-api.service
```

Comment:
In this case it's technically not needed but it's not harming anything either. If you prefer can I remove it so it's like the others.

Reply:
I'm more concerned that if we look at this in the future, we might think this code works different from the others because of the syntactic difference. If it's more consistent/standardized then we don't have to think as hard about the code.